tap_klaviyo/utils.py

```diff
@@ -185,3 +191,17 @@ def transfrom_and_write_records(events, stream):
                 transformer.transform(
                     event, event_schema, event_mdata
             ))
+
+# return the 'timeout'
+def get_request_timeout():
+    args = singer.utils.parse_args([])
+    # get the value of request timeout from config
+    config_request_timeout = args.config.get('request_timeout')
+
+    if config_request_timeout and float(config_request_timeout):
```

Comment:
Add detailed comment over if condition.

Reply:
I can't see the meaning of using the request_timeout variable.

Reply:
Updated the code and returned the value directly.